addons/hr_attendance/static/src/js/kiosk_mode.js

```diff
@@ -55,9 +56,17 @@ var KioskMode = AbstractAction.extend({
 
     destroy: function () {
         core.bus.off('barcode_scanned', this, this._onBarcodeScanned);
+        clearInterval(this._interval);
         clearInterval(this.clock_start);
         this._super.apply(this, arguments);
     },
+
+    call2Db: function () {
+        return this._rpc({
+               model: 'res.users',
+               method: 'search_read',
```

Comment:
usually, for this scenario, we use the route '/web/webclient/version_info'

Reply:
I need it in the hr_atttendance module wich does not depends on web

Reply:
Actually ``hr_attendance`` depends on ``hr`` that depends on ``web``.